.github/workflows/pr-check.yml

```
@@ -0,0 +1,225 @@
+    
+    - name: Generate coverage report
+      if: always()
+      run: |
+        if [ -f coverage/coverage-summary.json ]; then
+          COVERAGE=$(node -e "
+            const coverage = require('./coverage/coverage-summary.json');
+            const pct = coverage.total.statements.pct;
+            console.log(pct);
+          ")
+          echo "COVERAGE_PCT=$COVERAGE" >> $GITHUB_ENV
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)